Document wrangler r2 bucket lifecycle command (list, add, remove, set) (#18074)
cloudflare/cloudflare-docs: src/content/docs/r2/buckets/object-lifecycles.mdx

5. Fill out the fields for the new rule.
6. When you are done, select **Add rule**.

### Wrangler

> #### Set up Wrangler

Reviewer: ```suggestion

```

> To begin, install [`npm`](https://docs.npmjs.com/getting-started). Then [install Wrangler, the Developer Platform CLI](/workers/wrangler/install-and-update/).

Reviewer: ```suggestion
1. Install [`npm`](https://docs.npmjs.com/getting-started). 
2. Install [Wrangler, the Developer Platform CLI](/workers/wrangler/install-and-update/).
```

> #### Add a lifecycle rule

Reviewer: ```suggestion

```

> Log in to Wrangler with the [`wrangler login` command](/workers/wrangler/commands/#login). Then add a lifecycle rule to your bucket by running the [`r2 bucket lifecycle add` command](/workers/wrangler/commands/#lifecycle-add).

Reviewer: ```suggestion
3. Log in to Wrangler with the [`wrangler login` command](/workers/wrangler/commands/#login). 
4. Add a lifecycle rule to your bucket by running the [`r2 bucket lifecycle add` command](/workers/wrangler/commands/#lifecycle-add).
```

```sh
npx wrangler r2 bucket lifecycle add <BUCKET_NAME> [OPTIONS]
```

> #### Set the lifecycle configuration

Reviewer: ```suggestion

```

> Alternatively you can set the entire lifecycle configuration for a bucket from a JSON file using the [`r2 bucket lifecycle set` command](/workers/wrangler/commands/#lifecycle-set).

Reviewer: ```suggestion
5. Alternatively you can set the entire lifecycle configuration for a bucket from a JSON file using the [`r2 bucket lifecycle set` command](/workers/wrangler/commands/#lifecycle-set).
```
